specification/compute/resource-manager/Microsoft.Compute/GalleryRP/stable/2022-08-03/examples/galleryExamples/GalleryImageVersion_Create_WithVHD_UefiSettings.json

```diff
@@ -0,0 +1,348 @@
+{
+  "parameters": {
+    "subscriptionId": "{subscription-id}",
+    "resourceGroupName": "myResourceGroup",
+    "api-version": "2022-08-03",
+    "galleryName": "myGalleryName",
```

Comment:
You need to edit gallery.json to include a reference to this example file similar to this: 
https://github.com/Azure/azure-rest-api-specs/blob/edef848d91713d45656fbfb357a75ab96c2b98e6/specification/compute/resource-manager/Microsoft.Compute/GalleryRP/stable/2022-08-03/gallery.json#L628